ksql-execution/src/main/java/io/confluent/ksql/execution/plan/WindowedTableSource.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.execution.plan;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.confluent.ksql.execution.timestamp.TimestampColumn;
+import io.confluent.ksql.name.SourceName;
+import io.confluent.ksql.schema.ksql.LogicalSchema;
+import java.util.Optional;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.streams.Topology.AutoOffsetReset;
+import org.apache.kafka.streams.kstream.Windowed;
+
+public final class WindowedTableSource
```

Comment:
I'll probably merge https://github.com/confluentinc/ksql/pull/3970 before this goes in. Can you update this class to also include `WindowInfo`, as that PR does for `WindowedStreamSource`?